### PR TITLE
build: adds leverage liquidity cards

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Adds current APY to portfolio overview chart
- Updates leveraged liquidity cards
- Updates vault cards

### CODE REVIEW

File: packages/common/mui/src/lib/card-variants/incentive-active/incentive-active.tsx
Comment (jeffywu) [inline packages/common/mui/src/lib/card-variants/incentive-active/incentive-active.tsx:1]: This component should be called IncentiveLeveraged instead

File: packages/common/mui/src/lib/card-variants/incentive-active/incentive-active.tsx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+          }}
+        >
+          {buttonText}
+        </Button>
+      </Card>
+    </Link>
+  );
+};
+
+const ContentWrapper = styled(Box, {
```
Comment (jeffywu) [inline packages/common/mui/src/lib/card-variants/incentive-active/incentive-active.tsx:157]: Can we export these from some file and share them across the different cards? They look to be repeated in multiple files.

File: packages/features/portfolio/src/containers/portfolio-overview/hooks/use-totals-chart.tsx [hunk 2]
```diff
@@ -97,6 +104,20 @@ export const useTotalsChart = () => {
...
   }
 
+  if (accountApy) {
+    barConfig.push({
+      dataKey: 'currentApy',
+      title: <FormattedMessage defaultMessage="Current APY" />,
+      toolTipTitle: <FormattedMessage defaultMessage="Debts" />,
+      fill: 'transparent',
+      radius: [8, 8, 0, 0],
+      currencySymbol: FiatSymbols[baseCurrency]
```
Comment (jeffywu) [inline packages/features/portfolio/src/containers/portfolio-overview/hooks/use-totals-chart.tsx:114]: Assume that the currencySymbol does not actually have any effect here?

File: packages/common/mui/src/lib/card-variants/incentive-active/incentive-active.tsx (hunk 1 above)
Comment (matthew-garrett) [reply packages/common/mui/src/lib/card-variants/incentive-active/incentive-active.tsx:157]: They're slightly different and only used in the incentive leveraged and fixed cards

File: packages/features/portfolio/src/containers/portfolio-overview/hooks/use-totals-chart.tsx (hunk 2 above)
Comment (matthew-garrett) [reply packages/features/portfolio/src/containers/portfolio-overview/hooks/use-totals-chart.tsx:114]: Correct this was just to please the type

File: packages/common/mui/src/lib/card-variants/incentive-active/incentive-active.tsx (hunk 1 above)
Comment (matthew-garrett) [reply packages/common/mui/src/lib/card-variants/incentive-active/incentive-active.tsx]: I imported StyledIcon from the fixed card

**Pull request merged**